src/test/mochitest/browser_dbg-call-stack.js

```diff
@@ -22,9 +22,6 @@ add_task(async function() {
 
   toggleCallStack(dbg);
 
-  const notPaused = findElement(dbg, "callStackBody").innerText;
```

Comment:
we can change this test to be so that it passes if notPaused is not found
```javascript
const notPaused = findElement(dbg, "callStackBody");
is(!notPaused, true, "Call Stack body is not shown");
```